demisto_sdk/commands/validate/file_validator.py

```diff
@@ -523,13 +527,121 @@ def validate_committed_files(self):
+                self._is_valid = False
+
+        elif checked_type(file_path, YML_INTEGRATION_REGEXES) or file_type == 'integration':
+            integration_validator = IntegrationValidator(structure_validator)
+            if not integration_validator.is_valid_file(validate_rn=False):
+                self._is_valid = False
+
+        elif checked_type(file_path, PACKAGE_SCRIPTS_REGEXES) or file_type == 'script':
+            # Set file path to the yml file
+            structure_validator.file_path = file_path
+            script_validator = ScriptValidator(structure_validator)
+
+            if not script_validator.is_valid_file(validate_rn=False):
+                self._is_valid = False
+
+        elif re.match(BETA_INTEGRATION_REGEX, file_path, re.IGNORECASE) or \
+                re.match(BETA_INTEGRATION_YML_REGEX, file_path, re.IGNORECASE) or file_type == 'betaintegration':
+            integration_validator = IntegrationValidator(structure_validator)
+            if not integration_validator.is_valid_beta_integration():
+                self._is_valid = False
+
+        elif re.match(IMAGE_REGEX, file_path, re.IGNORECASE):
+            image_validator = ImageValidator(file_path)
+            if not image_validator.is_valid():
+                self._is_valid = False
+
+        # incident fields and indicator fields are using the same scheme.
+        elif checked_type(file_path, JSON_INDICATOR_AND_INCIDENT_FIELDS) or \
+                file_type in ('incidentfield', 'indicatorfield'):
+            incident_field_validator = IncidentFieldValidator(structure_validator)
+            if not incident_field_validator.is_valid_file(validate_rn=False):
+                self._is_valid = False
+
+        elif checked_type(file_path, [REPUTATION_REGEX]) or file_type == 'reputation':
+            reputation_validator = ReputationValidator(structure_validator)
+            if not reputation_validator.is_valid_file(validate_rn=False):
+                self._is_valid = False
+
+        elif checked_type(file_path, JSON_ALL_LAYOUT_REGEXES) or file_type == 'layout':
+            layout_validator = LayoutValidator(structure_validator)
+            if not layout_validator.is_valid_layout(validate_rn=False):
+                self._is_valid = False
+
+        elif checked_type(file_path, JSON_ALL_DASHBOARDS_REGEXES) or file_type == 'dashboard':
+            dashboard_validator = DashboardValidator(structure_validator)
+            if not dashboard_validator.is_valid_dashboard(validate_rn=False):
+                self._is_valid = False
+
+        elif checked_type(file_path, JSON_ALL_INCIDENT_TYPES_REGEXES) or file_type == 'incidenttype':
+            incident_type_validator = IncidentTypeValidator(structure_validator)
+            if not incident_type_validator.is_valid_incident_type(validate_rn=False):
+                self._is_valid = False
+
+        elif checked_type(file_path, CHECKED_TYPES_REGEXES):
+            pass
```

Comment:
when will you get here?

Reply:
Not sure, copied from validate_added_files, added a print here, so I'll see what happens